### PR TITLE
Added "Gitflow Operations Popup..."-Action

### PR DESCRIPTION
**Pull request by avbk**
Hey Opher,

first of all, thanks for your awesome work on this plugin. It's one of my most used plugins within the idea product family.
One thing I missed in this plugin was to open up the gitflow popup with a keyboard shortcut (in analogy to the "VCS Operations Popup..."). So I tried my best, to add this feature to your plugin ;-)

I hope you like and accept my work and include it in your next release.

Best regards,
Alex

### CODE REVIEW

File: src/gitflow/ui/GitflowWidget.java [hunk 1]
```diff
@@ -178,4 +193,43 @@ private static String getDisplayableBranchTooltip(GitRepository repo) {
...
+        return null;
+    }
+
+    /**
+     * Shows the action popup of this widget in the center of the provided frame. If there are no
+     * actions available for this widget, the popup will not be shown.
+     *
+     * @param frame The frame that will be used for display
+     */
+    public void showPopupInCenterOf(@NotNull JFrame frame) {
```
Comment (avbk) [inline src/gitflow/ui/GitflowWidget.java]: I am not sure, if the content of this function should be wrapped into some kind of ```ApplicationManager.getApplication().invokeLater(/* ... */);```? I did not fully get why you are running the update method asynchronously. I tried it both ways and could not see a difference. So I sticked with the easy solution ;-)

**Comment by OpherV** [comment]
Very cool!
I'll have a look soon and merge if it all checks out
thanks :)

**Comment by avbk** [comment]
Cool, so everything seems to be fine? Just out of curiosity, can you explain me the purpose of running the actions with ```invokeLater()```?

**Comment by OpherV** [comment]
Looks like it :)

Git/HgRepositories can be initialized before the widget is constructed so the widget wouldn't get the initial repo change event and wouldn't initialize. Doing the update async solves this and prevents the ui thread from becoming stuck on initialization.

**Comment by OpherV** [comment]
By the way your PR has now rolled out with the new version!

**Pull request merged**